modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -112,53 +110,57 @@ This command can take a few moments to run.
 ifdef::aws-sts[]
 . If your cluster uses cluster capabilities to disable one or more optional components, delete the `CredentialsRequest` custom resources for any disabled components.
 +
-.Example `credrequests` directory contents for {product-title} 4.12 on AWS
+.Example `credrequests` directory contents for {product-title} 4.13 on AWS
 +
 [source,terminal]
 ----
-0000_30_machine-api-operator_00_credentials-request.yaml <1>
-0000_50_cloud-credential-operator_05-iam-ro-credentialsrequest.yaml <2>
-0000_50_cluster-image-registry-operator_01-registry-credentials-request.yaml <3>
-0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml <4>
-0000_50_cluster-network-operator_02-cncc-credentials.yaml <5>
-0000_50_cluster-storage-operator_03_credentials_request_aws.yaml <6>
+0000_30_cluster-api_00_credentials-request.yaml <1>
+0000_30_machine-api-operator_00_credentials-request.yaml <2>
+0000_50_cloud-credential-operator_05-iam-ro-credentialsrequest.yaml <3>
+0000_50_cluster-image-registry-operator_01-registry-credentials-request.yaml <4>
+0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml <5>
+0000_50_cluster-network-operator_02-cncc-credentials.yaml <6>
+0000_50_cluster-storage-operator_03_credentials_request_aws.yaml <7>
 ----
 +
-<1> The Machine API Operator CR is required.
-<2> The Cloud Credential Operator CR is required.
-<3> The Image Registry Operator CR is required.
-<4> The Ingress Operator CR is required.
-<5> The Network Operator CR is required.
-<6> The Storage Operator CR is an optional component and might be disabled in your cluster.
+<1> The Cluster API Operator CR is required.
```

Comment:
Cluster API is tech preview and will only appear if `--enable-tech-preview` flag was passed. If the newer `--included` is used from `oc extract` command, then the cluster will have to also be in techpreview for this to appear.) Might consider leaving this one out for simplicity? I believe we decided similar when discussing if we should document `--enable-tech-preview`?

Reply:
I go back and forth on this (completeness vs. simplicity). We do mention this:
> If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.

So maybe keep the item but adjust the note?
```suggestion
<1> For clusters that use the `TechPreviewNoUpgrade` feature set, the Cluster API Operator CR is required.
```